kiwi/solver/repository/deb.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2015 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+# project
+from kiwi.solver.repository.base import SolverRepositoryBase
+
+
+class SolverRepositoryDeb(SolverRepositoryBase):
+    """
+    **Class for SAT solvable creation for apt-deb type repositories.**
+    """
+    def _setup_repository_metadata(self):
+        """
+        Download repo metadata for rpm-md specific repositories
```

Comment:
This is wrong, since this running against `apt-deb` repositories, not `rpm-md` ones.

Reply:
yes my bad, fixed it. Thanks